CMSIS/NN/Source/SoftmaxFunctions/arm_softmax_s8.c

```diff
@@ -28,10 +28,50 @@
  *
  * -------------------------------------------------------------------- */
 
-#include "arm_nnfunctions.h"
+#include "arm_nnsupportfunctions.h"
 
 #define ACCUM_BITS 12
 
+#ifdef ARM_MATH_MVEI
+int32x4_t arm_nn_exp_on_negative_values_32x4(int32x4_t val)
```

Comment:
Shouldn't this go in arm_nnsupportfunctions.h together with all the other quant ops?

Reply:
If it is used in more than one location/file, then yes. We can move it to the header file when that happens.